app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -196,13 +169,22 @@ public HighLevelProxy getProxy(URI host) {
             if(openSessions.containsKey(host)) {
                 return openSessions.get(host);
             } else {
-                HighLevelProxy proxy = ProtocolProxyFactory.getInstance().createHighLevelProxy(host, person);
+                HighLevelProxy proxy = ProtocolProxyFactory.getInstance().createHighLevelProxy(host, person, dataHandler);
                 openSessions.put(host, proxy);
                 return proxy;
             }
         }
     }
 
+    /**
+     * Add a new LAO to the app
+     *
+     * @param lao to add
+     */
+    public void createLao(Lao lao) {
```

Comment:
Why change the name from addLao to createLao? If it's only used when creating a LAO this comment can be ignored.

Reply:
I'm sure there was a reason... but whatever, I reversed it